app/models/collection.rb

```diff
@@ -198,21 +198,18 @@ def parts
   # @param key [String]
   # @return [Array] an array of array of {DataAssociation}s
   def data_matrix(key)
-
     pas = part_association_list
-    part_ids = pas.collect { |p| p.part_id }.uniq
 
-    das = DataAssociation.where(parent_class: 'Item', parent_id: part_ids, key: key)
+    part_ids = pas.collect { |p| p.part_id }.uniq
```

Comment:
Clarification question as I am getting familiar with rails again. Would
`part_ids = pas.collect { |p| p.part_id }.uniq` 
equivalent to
`part_ids = pas.collect(&:part_id).uniq` ?

Reply:
yes